dataprofiler/profilers/profiler_options.py

```diff
@@ -483,6 +484,19 @@ def __deepcopy__(self, memo):
                 setattr(result, k, copy.deepcopy(v, memo))
         return result
 
+    @property
+    def properties(self):
+        """
+        Returns a copy of the option properties.
+
+        :return: dictionary of the option's properties attr: value
+        :rtype: dict
+        """
+        props = {k: copy.deepcopy(v)
+                 for k,v in self.__dict__.items() if k != 'data_labeler_object'}
+        props['data_labeler_object'] = self.data_labeler_object
```

Comment:
I'm not 100% sure what would happen, but I'm guessing an error could happen here if you had multiple active instances of the data_labeler going on (i.e. multiple profiling). This could cause errors if the data_labeler_object was active; if it's not active it'll maybe be okay.

Reply:
I agree. However, the current code didn't allow setting or validation if a data labeler was set. 

Additionally, the previous code had the same issue where it was overwriting a mutable object, but this maintains the options having priority which may not be ideal.